civis/futures.py

```diff
@@ -110,14 +166,6 @@ def _check_message(self, message):
             return False
         return match
 
-    def _poll_and_set_api_result(self):
-        with self._condition:
-            try:
-                result = self.poller(*self.poller_args)
-                self._set_api_result(result)
-            except Exception as e:
-                self._set_api_exception(exc=e)
-
```

Comment:
Some `CivisFuture` clean-up that should've happened in #425. This `_poll_and_set_api_result` method is no longer used by anything since I've removed all the PubNub code.